README.md

```diff
@@ -1 +1,58 @@
-# nprintML
\ No newline at end of file
+# nprintML
+
+THIS IS WHAT NPRINTML IS!
```

Comment:
:writing_hand: Needn't be much – or even pretty – but I thought you @JordanHolland might like to supply a terse (or verbose) description here.

I'm sure it'll change in the future, but I'd prefer not to merge in anything too lame, nor any easter eggs.

Feel free to comment, add a changeset to the branch, or perhaps easiest just use the Github comment "insert a suggestion" feature (ctrl+g).